data/net/minecraft/client/gui/screens/ConfirmLinkScreen.mapping

```diff
@@ -2,30 +2,30 @@ CLASS net/minecraft/client/gui/screens/ConfirmLinkScreen
 	METHOD <init> (Lit/unimi/dsi/fastutil/booleans/BooleanConsumer;Ljava/lang/String;Z)V
 		ARG 1 callback
 		ARG 2 url
-		ARG 3 trustedLink
+		ARG 3 trusted
 	METHOD <init> (Lit/unimi/dsi/fastutil/booleans/BooleanConsumer;Lnet/minecraft/network/chat/Component;Ljava/lang/String;Lnet/minecraft/network/chat/Component;Z)V
 		ARG 1 callback
 		ARG 2 title
 		ARG 3 url
 		ARG 4 noButton
-		ARG 5 trustedLink
+		ARG 5 trusted
 	METHOD <init> (Lit/unimi/dsi/fastutil/booleans/BooleanConsumer;Lnet/minecraft/network/chat/Component;Ljava/lang/String;Z)V
 		ARG 1 callback
 		ARG 2 title
 		ARG 3 url
-		ARG 4 trustedLink
+		ARG 4 trusted
 	METHOD <init> (Lit/unimi/dsi/fastutil/booleans/BooleanConsumer;Lnet/minecraft/network/chat/Component;Lnet/minecraft/network/chat/Component;Ljava/lang/String;Lnet/minecraft/network/chat/Component;Z)V
 		ARG 1 callback
 		ARG 2 title
 		ARG 3 message
 		ARG 4 url
 		ARG 5 noButton
-		ARG 6 noWarning
+		ARG 6 trusted
 	METHOD confirmMessage (Z)Lnet/minecraft/network/chat/MutableComponent;
-		ARG 0 trustedLink
+		ARG 0 trusted
 	METHOD confirmMessage (ZLjava/lang/String;)Lnet/minecraft/network/chat/MutableComponent;
-		ARG 0 trustedLink
-		ARG 1 extraInfo
+		ARG 0 trusted
+		ARG 1 callback
```

Comment:
Something to investigate with an IDE again

Reply:
Yea once again no @PiTheGuy , This should be extraInfo since it's the string pased into and appended to the confirm message. In this case it's the "url" being passed in by third constructor ``ConfirmLinkScreen(BooleanConsumer, Component, String, Component, boolean)``